src/Umbraco.Cms.Api.Management/Factories/MemberPresentationFactory.cs

```diff
@@ -72,4 +80,29 @@ public IEnumerable<VariantItemResponseModel> CreateVariantsItemResponseModels(IM
 
     public MemberTypeReferenceResponseModel CreateMemberTypeReferenceResponseModel(IMemberEntitySlim entity)
         => _umbracoMapper.Map<MemberTypeReferenceResponseModel>(entity)!;
+
+    private async Task<MemberResponseModel> RemoveSensitiveDataAsync(IMember member, MemberResponseModel responseModel)
```

Comment:
I agree, it would be really good if we could remove those properties at service layer when fetching a member. That however is for another time - it requires immutable members to avoid loss of data.